src/sonic-dhcp-utilities/tests/test_data/mock_config_db.json

```diff
@@ -58,6 +64,17 @@
             "netmask": "255.255.255.0",
             "state": "enabled"
         },
+        "bridge_midplane": {
```

Comment:
Suggest have a new mock config db for smart switch and create new test cases

Reply:
Fixed